test/components-pages/fixtures/component-init-async/template.marko

```diff
@@ -1,5 +1,3 @@
-lasso-page dependencies=data.browserDependencies lasso=data.lasso
-
 import Promise from 'promise-polyfill';
```

Comment:
I notice you [upgraded this module](https://github.com/marko-js/marko/pull/989/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R101).  Is it even needed?

Reply:
Yeah I think you are right. I thought I saw a test that was testing specifically for a poly filled promise but I guess I was wrong. I updated to PR to remove this dependency.